consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,43 @@
-# Create your models here.
 from django.db import models
```

Comment:
Do we want some UUID ID fields on these? (setting in `settings.py` didn't work: https://github.com/i-dot-ai/consultation-analyser/commit/0b29bb5dbb00e4ca7327909fd5637ae6e00ba656)

Reply:
Now updated for all models including the `Question` model.

Given that we're only working locally, I've deleted existing migrations and started again.